### PR TITLE
Merge the two User Manuals into one.

### PR DESCRIPTION
**Pull request by solemabrothers**

### CODE REVIEW

File: SUMMARY.md [hunk 1]
```diff
@@ -80,7 +81,17 @@
   
   * [3.0.4](release-notes/ugandaemr-304.md)
   
-  * [3.1.0](release-notes/ugandaemr-310.md)
+  * [3.1.0](release-notes/ugandaemr-310.md) 
+ 
+* UgandaEMR Mobile
```
Comment (ssmusoke) [inline SUMMARY.md]: @solemabrothers move the section above the release notes which should be the last

File: ugandaemr_mobile/SUMMARY.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Table of contents
```
Comment (ssmusoke) [inline ugandaemr_mobile/SUMMARY.md]: @solemabrothers This file is not needed anymore

File: ugandaemr_mobile/README.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Introduction
```
Comment (ssmusoke) [inline ugandaemr_mobile/README.md:1]: @solemabrothers Add the download, Installation and login content to reduce the number of pages

File: ugandaemr_mobile/assess-patients.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# Work Summary
```
Comment (ssmusoke) [inline ugandaemr_mobile/assess-patients.md]: @solemabrothers I think you should combine, Download Patients, Find Patients, Work Summary, Drug Dispensing into a single file - I do not see sufficient content for 3 pages

Comment (ssmusoke) [reply ugandaemr_mobile/assess-patients.md]: @solemabrothers The first heading should always be a heading 2 (##) not heading 1

File: SUMMARY.md (hunk 1 above)
Comment (solemabrothers) [reply SUMMARY.md]: Done

File: ugandaemr_mobile/uninstalling-ugandaemr-mobile.md [hunk 5]
```diff
@@ -0,0 +1,2 @@
+# Uninstalling UgandaEMR Mobile
```
Comment (ssmusoke) [inline ugandaemr_mobile/uninstalling-ugandaemr-mobile.md]: @solemabrothers This page is empty - delete it

File: ugandaemr_mobile/uninstall-ugandaemr-mobile.md [hunk 6]
```diff
@@ -0,0 +1,7 @@
+# Uninstalling UgandaEMR Mobile
```
Comment (ssmusoke) [inline ugandaemr_mobile/uninstall-ugandaemr-mobile.md]: @solemabrothers Move this to the bottom of the installation screen

File: ugandaemr_mobile/terminologies.md [hunk 7]
```diff
@@ -0,0 +1,12 @@
+# Terminologies
```
Comment (ssmusoke) [inline ugandaemr_mobile/terminologies.md]: @solemabrothers This page is not needed as it is available in the UgandaEMR main user manual

File: ugandaemr_mobile/SUMMARY.md (hunk 2 above)
Comment (solemabrothers) [reply ugandaemr_mobile/SUMMARY.md]: Done

File: ugandaemr_mobile/uninstalling-ugandaemr-mobile.md (hunk 5 above)
Comment (solemabrothers) [reply ugandaemr_mobile/uninstalling-ugandaemr-mobile.md]: Done

File: ugandaemr_mobile/terminologies.md (hunk 7 above)
Comment (solemabrothers) [reply ugandaemr_mobile/terminologies.md]: Done

File: ugandaemr_mobile/drug-administration.md [hunk 8]
```diff
@@ -0,0 +1,14 @@
+# Drug Dispensing
+
+1. The screen belows shows how to administer drug, each patients current Regimen is displayed as shown in the figure below.
+
+   ![Drug Screen](.gitbook/assets/img3.jpg)
+
+4.Tap on the drug or combination from the Drug administration tables as shown below to enter pills dispensed and duration in days. ![Drug administration Screen](.gitbook/assets/img4.jpg)
```
Comment (ssmusoke) [inline ugandaemr_mobile/drug-administration.md]: @solemabrothers Number the items in this list properly and start sentences with Capital Letters

File: ugandaemr_mobile/README.md (hunk 3 above)
Comment (solemabrothers) [reply ugandaemr_mobile/README.md:1]: Done

File: ugandaemr_mobile/assess-patients.md (hunk 4 above)
Comment (solemabrothers) [reply ugandaemr_mobile/assess-patients.md]: Done

File: ugandaemr_mobile/uninstall-ugandaemr-mobile.md (hunk 6 above)
Comment (solemabrothers) [reply ugandaemr_mobile/uninstall-ugandaemr-mobile.md]: Done

File: ugandaemr_mobile/terminologies.md (hunk 7 above)
Comment (solemabrothers) [reply ugandaemr_mobile/terminologies.md]: Done

File: ugandaemr_mobile/drug-administration.md (hunk 8 above)
Comment (solemabrothers) [reply ugandaemr_mobile/drug-administration.md]: Done

File: ugandaemr_mobile/download-patients.md [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+
+## Work Summary
+1. To assess a patient in UgandaEMR mobile app Tap Work Summary from the menu screen or home screen, Select visit date for patients with appointments. use the filter to navigate through different options. ![work summary Screen](assets/img15.jpg)
+2. On selecting the visit date is selected, alist of downloaded patients will be seen as shown in the figure below. to assess a patient Tap on the patient record. ![Assess Patient button](.gitbook/assets/img16.jpg)
+3. To assess a patient Tap Assess Patient to start an assessment as shown ![Assessment Screen](images/client_selected.png)
+  NOTE:
+   If the patient is  represented Tap Yes button and Dispense drugs to the Patient otherwise click YES to start the assessment.
+   start assessing the patient. 
+## Drug Dispensing 
+1 . Tap on the drug or combination from the Drug administration tables as shown below to enter pills dispensed and duration in days. ![Drug administration Screen](images/drug_dispensing.png)
```
Comment (ssmusoke) [inline ugandaemr_mobile/download-patients.md]: @solemabrothers Remove the empty spaces between the number and the dots, that is special meaning to that

File: ugandaemr_mobile/README.md (hunk 3 above)
Comment (ssmusoke) [reply ugandaemr_mobile/README.md:1]: @solemabrothers The Introduction has to be a heading 2 so other headings go a level lower

**Pull request merged**